engine/resource/src/resource_mounts.cpp

```diff
@@ -587,6 +589,15 @@ static dmResource::Result GetDependenciesInternal(DependencyIterContext* ctx, co
         if (dmResource::RESULT_RESOURCE_NOT_FOUND == result)
             continue;
 
+        if (include_requested_url) {
+            // include only once
+            include_requested_url = false;
+            dependencies.SetCapacity(dependencies.Capacity() + 1);
+            int size = dependencies.Size();
+            dependencies.SetSize(size + 1);
+            dependencies[size] = url_hash;
+        }
```

Comment:
Using this option, the requested URL will be included into the results, but it should be included only onceinclude_requested_url